app/herdbook.py

```diff
@@ -214,7 +214,7 @@ def herd(h_id):
     data = da.get_herd(h_id, session.get("user_id", None))
     return jsonify(data)
 
-@APP.route("/api/individual/<int:i_id>")
+@APP.route("/api/individual/<i_id>")
 @login_required
```

Comment:
I have now removed the cached from the decorators in my branch since it is not considering different users.  Do the same here, to test the last version of your code without the cache interfering.
`-@cache.cached(timeout=36000)`